src/components/slide-toggle/slide-toggle.spec.ts

```diff
@@ -0,0 +1,102 @@
+import {
+  it,
+  describe,
+  expect,
+  beforeEach,
+  inject,
+} from '@angular/core/testing';
+import {TestComponentBuilder} from '@angular/compiler/testing';
+import {By} from '@angular/platform-browser';
+import {Component} from '@angular/core';
+import {MdSlideToggle} from './slide-toggle';
+
+export function main() {
+  describe('MdSlideToggle', () => {
+    let builder: TestComponentBuilder;
+
+    beforeEach(inject([TestComponentBuilder], (tcb: TestComponentBuilder) => {
+      builder = tcb;
+    }));
+
+    it('should update the model correctly', (done: () => void) => {

```

Comment:
Rather than using the `done` argument for these tests, you should be able to use the new `async()` function that will complete the test automatically when the async operations are done. You can see the checkbox tests as an example.